### PR TITLE
add description returning method on MySQL and SQlite

### PR DESCRIPTION
**Pull request by HurSungYun**
if my grammar is wrong, please let me know

### CODE REVIEW

File: sections/builder.js [hunk 1]
```diff
@@ -1331,7 +1331,7 @@ export default [
...
-    description: "Utilized by PostgreSQL, MSSQL, and Oracle databases, the returning method specifies which column should be returned by the insert and update methods. Passed column parameter may be a string or an array of strings. When passed in a string, makes the SQL result be reported as an array of values from the specified column. When passed in an array of strings, makes the SQL result be reported as an array of objects, each containing a single property for each of the specified columns. The returning method is not supported on Amazon Redshift.",
+    description: "Utilized by PostgreSQL, MSSQL, and Oracle databases, the returning method specifies which column should be returned by the insert and update methods. Passed column parameter may be a string or an array of strings. When passed in a string, makes the SQL result be reported as an array of values from the specified column. When passed in an array of strings, makes the SQL result be reported as an array of objects, each containing a single property for each of the specified columns. The returning method is not supported on Amazon Redshift or case of multiple inserts on MySQL and SQlite",
```
Comment (kibertoad) [inline sections/builder.js]: I would suggest wrapping 'returning' in `\"`

`When passed in an array of strings` - not sure about syntax here, shouldn't it be "When an array of strings is passed in"?

`makes the SQL result be reported` -> "... to be reported"?

`The returning method is not supported on Amazon Redshift or case of multiple inserts on MySQL and SQlite"` -> "is entirely unsupported on Amazon Redshift and has no support for multiple inserts on MySQL and SQLite"?

**Comment by kibertoad** [comment]
@HurSungYun One more tiny thing - final sentence originally ended with "." - maybe it makes sense to put it there as well now?

**Comment by kibertoad** [comment]
Thank you!

**Comment by elhigu** [comment]
This actually added an invalid bit of information, since it says that returning doesn't work for mysql for multiple inserts. But actually it just does not work for mysql at all and I suppose neither for sqlite.

**Comment by elhigu** [comment]
Reverting this one.

**Pull request merged**